tutor/resources/styles/components/questions-dashboard.less

```diff
@@ -58,6 +58,8 @@
   .sections-list {
     padding: @padding;
     background-color: @tutor-white;
+    overflow: auto;
+    height: ~"calc(100vh - 250px)";
   }
```

Comment:
should this (and rule below) be max-height?   What happens if we have a very small book, should it still expand to have footer at bottom?